pyproject.toml

```diff
@@ -19,19 +19,19 @@ requires-python = ">=3.8"
 dynamic = ["version"]
 
 dependencies = [
-    "qiskit==0.36.0",
-    "pytket==1.2.2",
-    "pytket-qiskit==0.25.0",
-    "pandas==1.3.5",
-    "flask==2.2.2",
-    "networkx==2.8.6",
-    "pytest==7.1.3",
-    "qiskit_finance==0.3.4",
-    "qiskit_machine-learning==0.4.0",
-    "qiskit-nature[pyscf]==0.3.1",
-    "qiskit_optimization==0.3.2",
-    "packaging==21.3",
-    "tqdm==4.64.1",
+    "qiskit~=0.39.2",
+    "pytket~=1.9.0",
+    "pytket-qiskit~=0.30.0",
+    "pandas~=1.5.2",
+    "flask~=2.2.2",
+    "networkx~=2.8.8",
+    "pytest~=7.2.0",
```

Comment:
These non-essential dependencies can be even less strict. e.g. `pytest>=7` is totally sufficient.
Typically, I'd just go for the lowest possible version that works with the current code base (probably derived from a mixture of the qiskit dependencies) and not specify an upper cap.
For a rather lengthy treatment on version capping in Python, see https://iscinumpy.dev/post/bound-version-constraints/

Reply:
Done.